<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(59,68,75);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget4_2" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>340</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Precision:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="precision">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget4_3" native="true">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>380</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Recall:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="recall">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget4_4" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>420</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Accuracy:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="accuracy">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget4_5" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>460</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>FMeasure:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fMeasure">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="controlsFrame">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>201</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid white;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string>Controls</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="TrainButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>520</y>
       <width>149</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#TrainButton
{
color:rgb(255, 255, 255);
border:solid;
border-radius:10px;
border-color: rgb(255,255,255);
border-width:2px;
}
QPushButton#TrainButton:checked { background: rgb(83, 84, 85); }
QPushButton#TrainButton:pressed { background: rgb(83, 84, 85); }</string>
     </property>
     <property name="text">
      <string>TRAIN</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>401</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid white;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="PlotArea" name="PlotWidget" native="true">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>30</y>
       <width>401</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string>Statistics</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="widget4_6" native="true">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>490</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);
border:none;</string>
        </property>
        <property name="text">
         <string>Time(s):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="time">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid white;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="clearButton_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>350</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#clearButton_2
{
color:rgb(255, 255, 255);
border:solid;
border-radius:10px;
border-color: rgb(255,255,255);
border-width:2px;
}
QPushButton#clearButton_2:checked { background: rgb(83, 84, 85); }
QPushButton#clearButton_2:pressed { background: rgb(83, 84, 85); }</string>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="DrawScene" name="drawArea" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>280</width>
       <height>280</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string>Prediction</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="widget4" native="true">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>390</y>
       <width>181</width>
       <height>87</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>Predict:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineAnswer">
        <property name="font">
         <font>
          <pointsize>50</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);
border:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="SavePhotoButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>510</y>
       <width>125</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#SavePhotoButton
{
color:rgb(255, 255, 255);
border:solid;
border-radius:10px;
border-color: rgb(255,255,255);
border-width:2px;
margin-top:10px;
}
QPushButton#SavePhotoButton:checked { background: rgb(83, 84, 85); }
QPushButton#SavePhotoButton:pressed { background: rgb(83, 84, 85); }</string>
     </property>
     <property name="text">
      <string>Save Photo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LoadPhotoButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>478</y>
       <width>125</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#LoadPhotoButton
{
color:rgb(255, 255, 255);
border:solid;
border-radius:10px;
border-color: rgb(255,255,255);
border-width:2px;
margin-top:10px;
}
QPushButton#LoadPhotoButton:checked { background: rgb(83, 84, 85); }
QPushButton#LoadPhotoButton:pressed { background: rgb(83, 84, 85); }</string>
     </property>
     <property name="text">
      <string>Load Photo</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="controlElements" native="true">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>50</y>
      <width>173</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);
gridline-color: rgb(255, 255, 255);</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="0" colspan="2">
      <widget class="QWidget" name="widget3" native="true">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Cross Validation:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCheckBox" name="checkCrossValBox">
          <property name="text">
           <string>Enable</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Coef.</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QSpinBox" name="crossValBox">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Epochs</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QPushButton" name="TestButton">
       <property name="styleSheet">
        <string notr="true">QPushButton#TestButton
{
color:rgb(255, 255, 255);
border:solid;
border-radius:10px;
border-color: rgb(255,255,255);
border-width:2px;
}
QPushButton#TestButton:checked { background: rgb(83, 84, 85); }
QPushButton#TestButton:pressed { background: rgb(83, 84, 85); }</string>
       </property>
       <property name="text">
        <string>TEST</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Sample</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QSpinBox" name="epochsBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QWidget" name="widget1" native="true">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Layers</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="matrixButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Matrix</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoRepeat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QSpinBox" name="layerBox">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="graphButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Graph</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoRepeat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Implementation:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QWidget" name="widget2" native="true">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="text">
           <string>Weights:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="LoadButton">
          <property name="styleSheet">
           <string notr="true">QPushButton#LoadButton
{
color:rgb(255, 255, 255);
border:solid;
border-radius:10px;
border-color: rgb(255,255,255);
border-width:2px;
margin-top:10px;
}
QPushButton#LoadButton:checked { background: rgb(83, 84, 85); }
QPushButton#LoadButton:pressed { background: rgb(83, 84, 85); }</string>
          </property>
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="SaveButton">
          <property name="styleSheet">
           <string notr="true">QPushButton#SaveButton
{
color:rgb(255, 255, 255);
border:solid;
border-radius:10px;
border-color: rgb(255,255,255);
border-width:2px;
margin-top:10px;
}
QPushButton#SaveButton:checked { background: rgb(83, 84, 85); }
QPushButton#SaveButton:pressed { background: rgb(83, 84, 85); }</string>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QDoubleSpinBox" name="testBox">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>frame</zorder>
   <zorder>frame_2</zorder>
   <zorder>controlsFrame</zorder>
   <zorder>widget4_2</zorder>
   <zorder>widget4_3</zorder>
   <zorder>widget4_4</zorder>
   <zorder>widget4_5</zorder>
   <zorder>controlElements</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawScene</class>
   <extends>QWidget</extends>
   <header location="global">../src/NerualNetworkMLP/View/DrawScene.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlotArea</class>
   <extends>QWidget</extends>
   <header location="global">../src/NerualNetworkMLP/View/PlotArea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
